src/linear_algebra/linear_pmap.lean

```diff
@@ -322,19 +322,67 @@ instance : has_smul M (E →ₗ.[R] F) :=
   { domain := f.domain,
     to_fun := a • f.to_fun }⟩
 
-lemma smul_apply (a : M) (f : E →ₗ.[R] F) (x : ((a • f).domain)) :
+@[simp] lemma smul_apply (a : M) (f : E →ₗ.[R] F) (x : ((a • f).domain)) :
```

Comment:
With the earlier setup, `lean` was able to simplify an unapplied `⇑(a • f)`. Any reason to drop this?

Reply:
I had some issues with the `simp`-linters. Looking now into `linear_map`, I guess the correct thing is to make `smul_apply` simp and keep `coe_smul` as a non-simp lemma.

Reply:
I tried to move `@[simp]` attrs back. Let's see if it works. I also added one more `ext'` lemma that is useful if `f.domain = g.domain` is a `rfl`.